src/modes/intraOrderbook.ts

```diff
@@ -293,6 +293,9 @@ export async function findOpp({
                     orderPairObject.takeOrders[0].takeOrder.order.owner.toLowerCase() &&
                 // only orders that (priceA x priceB < 1) can be profitbale
                 v.quote!.ratio.mul(orderPairObject.takeOrders[0].quote!.ratio).div(ONE).lt(ONE),
+        )
+        .sort((a, b) =>
```

Comment:
is this change supposed to be on this PR?

Reply:
yes ofc